src/modules/commands/fight.js

```diff
@@ -0,0 +1,22 @@
+const isEmpty = require('lodash/isEmpty');
+const random = require('lodash/sample');
+const command = require('../command.filter');
+
+const fights = async function (request, context) {
+    const { i18n, send } = context;
+    const { args: opponent } = request;
+
+    if (isEmpty(opponent)) {
+        send(i18n('fight.userisnotdefined'));
+
+        return request;
+    }
+    const fighters = ['you', opponent];
```

Comment:
```suggestion
    const fighters = [user.id, opponentId];
```

наверное мы помещаем и управляемся именно рандомом пользователей?

Reply:
а то сейчас какой-то мусор, между you и каким-то обьектом